app/cli/cmd/attestation_add.go

```diff
@@ -90,30 +86,17 @@ func newAttestationAddCmd() *cobra.Command {
 
 			// In some cases, the attestation state is stored remotely. To control concurrency we use
 			// optimistic locking. We retry the operation if the state has changed since we last read it.
-			return backoff.RetryNotify(
```

Comment:
I extracted this logic to a common function and extended it with transient API errors